nbs/03_matcher.ipynb

```diff
@@ -34,25 +34,27 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        - Constant value checks (ensures that constant values match).
</pre><p>saying that you expect a dict is not enough, you have to say what schema you expect in the dict</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Added in Args: </p><p>        pattern_attrs (dict: attr_name -&gt; (attr_type, attr_value)): The pattern attributes.</p><p>        input_attrs (dict: attr_name -&gt; (attr_type, attr_value)): The input attributes.</p><p>is this enough?</p>

Reply:
<p>it is enough</p>